Payments/UserInterface/01a_ListAuthorizations.md

```diff
@@ -0,0 +1,299 @@
+
+        - ![Delete](../../Assets/Icons/Trash01.png "[Delete]") delete  
+            Click this entry to delete the selected view. A confirmation window to confirm the deletion is displayed. This menu entry is only displayed if you have selected a view.
+ 
+
+- *Connections*   
+    Click the drop-down list to select a connection. The following options are available:
+    - **All connections**  
+        Select this option to display authorization transactions for all configured connections. 
+    - **Connection name**  
+        Select this option to display authorization transactions for the selected connection. When a specific connection has been selected, the **All connections** option is no longer displayed. Click the selected connection again to deselect it.  
+
+
+
+- ![Search](../../Assets/Icons/Search.png "[Search]") (Search)   
+    Click this button to display the search bar and search for authorization transactions.   
+
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+    Click this button to update the list of authorization transactions.   
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+    Click this button to display the columns bar and to customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.   
+    > [Info] It is recommended to add the following columns to your view to get the required information and to better filter and sort the entries: Status ID, Status information (if applicable).    
+
+- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
+    Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active.  
+
+- [x]     
+    Select the checkbox to the left of one list entry to display the editing toolbar for this single entry.   
+    If you click the checkbox in the header, all entries in the list are selected. The editing toolbar shows all actions you can perform on several entries.
+
+
+- [CAPTURE]   
+   Click this button to capture the transaction manually, so that the order process can be continued. For detailed information, see [Capture authorization transaction](../Operation/01_ManageAuthorizations.md#capture-authorization-transaction).
+
+- [CANCEL]   
+   Click this button to cancel the transaction manually, so that the order is cancelled both at Actindo and at the payment service provider. For detailed information, see [Cancel authorization transaction](../Operation/01_ManageAuthorizations.md#cancel-authorization-transaction).
+
+- [VOID]   
+  Click this button to disable the transaction for the Actindo database. For detailed information, see [Void authorization transaction](../Operation/01_ManageAuthorizations.md#void-authorization-transaction).
```

Comment:
disable -> invalidate ?

Zutreffende Änderungen in Terminologie bei anderen Kapiteln (Integration/Operation) bitte durchgängig vornehmen.

Reply:
Done